run_server.py

```diff
@@ -2,9 +2,17 @@
 # uses wsgi to start up, and bypasses this file. So we are free to have debug
 # settings enabled.
 from Donut import app, config
+import argparse
+
+parser = argparse.ArgumentParser(description='Set up development server.')
+parser.add_argument('port_num', metavar='port_num', type=int, 
+                    help='Port on which to run server.')
+
+args = parser.parse_args()
+
 
 if __name__ == "__main__":
-    port = getattr(config, 'PORT', 5050)
+    port = getattr(config, 'PORT', args.port_num)
```

Comment:
Presumably if you pass in a port, you really want to use that port. `getattr(config, 'PORT', args.port_num)` will always use `config.PORT` unless that constant is not set, in which case it uses the one passed in by a flag.